editor.planx.uk/src/@planx/components/Pay/Public/InviteToPayForm.tsx

```diff
@@ -56,6 +57,17 @@ const StyledForm = styled("form")(({ theme }) => ({
   gap: theme.spacing(3),
 }));
 
+const StyledWarningBox = styled(Box)(({ theme }) => ({
```

Comment:
nit: I imagine we'll be using this again in other parts of @ianjon3s' redesign - could we make it a component that lives in `editor.planx.uk/src/ui`?

Reply:
yep this is a fair shout - moving over now :+1: I _think_ `editor.planx.uk/@planx/components/shared/Preview/` is actually the more fitting location based on other styled container-like components there (but the difference of this subdirectory to `src/ui` has always been a bit vague for me!)